edunext_openedx_extensions/edunext/management/commands/import_xml_db.py

```diff
@@ -3,28 +3,47 @@
 """
 Django command to import the users of an external database
 """
-from django.core.management.base import BaseCommand, CommandError
-from xml_importer.database_xml_dump_importer import DatabaseXmlDumpImporter
-from pprint import pformat as pf
 import json
 import logging
+from pprint import pformat as pf
 
-logger = logging.getLogger(__name__)
+from django.core.management.base import BaseCommand, CommandError
+
+try:
+    from xml_importer.database_xml_dump_importer import DatabaseXmlDumpImporter
+    FLAG_IMPORT = True
```

Comment:
Do we really need this, I understand we need the try on the import to run the class on the tests, but if the flag here is false, the code is going to fail either way after line 88. I suggest we leave this file as similar to the original as we can